lib/active_attr/version.rb

```diff
@@ -1,5 +1,5 @@
 module ActiveAttr
   # Complete version string
   # @since 0.1.0
-  VERSION = "0.8.5"
+  VERSION = "0.8.6"
```

Comment:
[PLEASE REMOVE] Never version bump in a PR. That's a job for maintainers only, and not appropriate within a PR, because you never know when the PR will be merged.

Reply:
done.